Angular InstantSearch/geo-search/src/styles.css

```diff
@@ -1,11 +1,3 @@
 /* You can add global styles to this file, and also import other style files */
-body,
-h1 {
-  margin: 0;
-  padding: 0;
-}
-
-body {
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
```

Comment:
in the just-merged PR by @dhayab there's default css + the css installed via cdn. Which is the correct way?

Reply:
fixed here: 7dd2c94b92aa9acb3d1ab3142b4b6d501676996c